### ISSUE TITLE
Edit World Name Easilly Using Flux

### ISSUE THREAD

**Issue by AmasterAmaster** [Yellow-Dog-Man/Resonite-Issues#3172]
### Is your feature request related to a problem? Please describe.

Currently, there are no obvious or easy solutions to edit a world name using flux. Especially for headless worlds that need to update relevant information. The only solutions to this requires userspace (or worse, ref hacking), which is unfortunate, as it would be nice to have the name dynamically change according to what happens within the world.

### Describe the solution you'd like

A ProtoFlux node specifically made for editing/updating a world name (optionally with proper permissions).

### Describe alternatives you've considered

Here are some things I considered and have tried:
- Setting the name manually using the session tab (this is impossible for headless worlds that need to update information, and when I am unable to be in the world when the world state changes)
- Ref hacking (I tried this and failed after finding out that you need to be in userspace to edit this value)
- Trying to get the headless's dash/userspace to edit the value (how does anyone do this???? Also not a user friendly solution at all)

### Additional Context

Here are some examples this node can help with:
- A theater world (especially headlesses) that can update the world name to show what is currently playing in the world.
- Score keeping in game worlds.
- Classroom worlds (especially headlesses) that can update the lesson as a world name, without wasting the teacher's time.

### Requesters

- AmasterAmaster

**Comment by Redd56** [comment]
would love this feature esp since the hidden star theater is now ready for normal use. as without this feature we are currently just using a refhack to edit the world info component (static id) which is very much not what we would like to do. esp since only admins can change the world name and thusly anyone who presses the button who isnt admin gets kicked out of the world.

**Comment by Redd56** [comment]
other workarrounds ive thought of include a python script that uses webhooks to send commands to the headless. Issue being you need to know python to make that and you need to impliment security safe webhooks AND to top it off not be connected to the headless.

a mod: im not good at modding and modding a headless makes it unstable. which for a headless is bad.

**Comment by AmasterAmaster** [comment]
Although this may be blocked by Hard Permissions -> Component Access, it would be very nice to have, noted above in the additional context section. But until this, the current implementation we have in the theater will have to do (or maybe I can further experiment to see if there is an automated way to set a world name in flux. Here is hoping.)
